### ISSUE TITLE
The weights for ynh_script_progression are super weird and probably don't make sense

### ISSUE THREAD

**Issue by alexAubin** [YunoHost-Apps/yeswiki_ynh#12]
Not the biggest issue ever, but if you look at the weights in ynh_script_progression they seem to be a sum() of the previous weight + the weight of the current step ... instead, what should happen is to weight each steps individually...

**Comment by Plumf** [comment]
I'm not sur to understand, could you explain this problem ? I remember that we have already exchanged about ynh_script_progression but I can't find our discussion on the forum

**Comment by alexAubin** [comment]
The currnet script progressions items are : 

```
ynh_script_progression --message="Validating installation parameters..." --weight=1
ynh_script_progression --message="Storing installation settings..." --weight=3
ynh_script_progression --message="Creating a MySQL database..." --weight=16
ynh_script_progression --message="Setting up source files..." --weight=17
ynh_script_progression --message="Configuring NGINX web server..." --weight=34
ynh_script_progression --message="Configuring system user..." --weight=36
ynh_script_progression --message="Configuring PHP-FPM..." --weight=40
ynh_script_progression --message="Configuring SSOwat..." --weight=42
ynh_script_progression --message="Finalizing installation..." --weight=44
ynh_script_progression --message="Configuring SSOwat..." --weight=47
ynh_script_progression --message="Reloading NGINX web server..." --weight=49
```

Each weight is higher than the previous one which seems super funky ... For example the first step (validating install parameters) has a weight of 1, and the last step (reloading nginx) has a weight of 49 ... but there's no way that reloading nginx takes 49 times longer than validating the install parameters...

**Comment by alexAubin** [comment]
As said it's not the biggest issue ever, but it also means that the resulting progress bar com

**Comment by alexAubin** [comment]
(Woopsies wrong button sorry)

**Comment by Plumf** [comment]
Solved by PR #18
